projects/swimlane/ngx-ui/src/lib/components/input/input.component.ts

```diff
@@ -24,7 +24,7 @@ import {
 } from '@angular/forms';
 import { BehaviorSubject } from 'rxjs';
 
-import { coerceBoolean } from '@swimlane/ngx-ui/utils';
```

Comment:
Looks like this function is exactly the same as `coerceBooleanProperty`. Should we remove this and replace with `coerceBooleanProperty` in other components that also use that?

Reply:
yes, going to open a PR to remove our internal util, this was only made because something was originally preventing us from referencing the cdk version.